pandas/tests/frame/test_analytics.py

```diff
@@ -738,6 +738,26 @@ def test_sem(self):
             self.assertFalse((result < 0).any())
             nanops._USE_BOTTLENECK = True
 
+    def test_sort_invalid_kwargs(self):
+        df = DataFrame([1, 2, 3], columns=['a'])
+
+        msg = "sort\(\) got an unexpected keyword argument 'foo'"
+        tm.assertRaisesRegexp(TypeError, msg, df.sort, foo=2)
+
```

Comment:
Correct, because `sort_values` does not take a `kwargs` parameter.  If you call `np.sort` on a `DataFrame`, it will call `DataFrame.sort` first, and that's where I want the `kwargs` validation to take place.